docs/api/en/constants/Renderer.html

```diff
@@ -9,6 +9,12 @@
 	<body>
 		<h1>WebGLRenderer Constants</h1>
 
+		<h2>GLSL Version</h2>
+		<code>
+		THREE.GLSL1
+		THREE.GLSL3
+		</code>
+
```

Comment:
Please move these constants to `Material Constants`. They are unrelated to `WebGLRenderer`.

Reply:
Never mind. I'll fix it on my side 👍 .